digital_land/pipeline.py

```diff
@@ -194,13 +203,20 @@ def load_redirect_lookup(self):
             old_entity = row.get("old-entity", "")
             entity = row.get("entity", "")
             status = row.get("status", "")
-            if old_entity and status:
+            if old_entity and entity and status:
```

Comment:
is this on purpose? I think if the status is 410 then there doesn't need to be an entity

Reply:
Had some merge issues and was added in error. Corrected now